src/vos/tests/vos_tests.c

```diff
@@ -269,6 +284,8 @@ main(int argc, char **argv)
 		case 'S':
 		case 'f':
 		case 'e':
+		case FORCE_CSUM:
```

Comment:
(style)  Possible switch case/default not preceeded by break or fallthrough comment